src/renderer/main/package.ts

```diff
@@ -1218,57 +1279,81 @@ function listFilter(column, btns, btn) {
  *
  * @param {string} instPath - An installation path.
  */
-async function displayNicommonsIdList(instPath) {
+async function displayNicommonsIdList(instPath: string) {
   const packages = await getPackages(instPath);
+  type PackageItemWithNicommonsId = {
+    info: {
+      name: string;
+      developer: string;
+      originalDeveloper?: string;
+      nicommons: string;
+    };
+    type: string[];
+  };
   const packagesWithNicommonsId = [
     {
       info: { name: 'AviUtl', developer: 'KENくん', nicommons: 'im1696493' },
-      type: [],
-    },
+      type: [] as never[],
+    } as PackageItemWithNicommonsId,
```

Comment:
```suggestion
    },
```

It may be better not to put `as` on objects that are entered manually.